<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>MainWindow</class>
 <widget class="QMainWindow" name="MainWindow">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>1366</width>
    <height>768</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>MainWindow</string>
  </property>
  <widget class="QWidget" name="centralwidget">
   <widget class="QStackedWidget" name="stackedWidget">
    <property name="geometry">
     <rect>
      <x>0</x>
      <y>0</y>
      <width>1366</width>
      <height>768</height>
     </rect>
    </property>
    <property name="currentIndex">
     <number>0</number>
    </property>
    <widget class="QWidget" name="page">
     <widget class="QWidget" name="horizontalLayoutWidget_3">
      <property name="geometry">
       <rect>
        <x>300</x>
        <y>30</y>
        <width>611</width>
        <height>401</height>
       </rect>
      </property>
      <layout class="QHBoxLayout" name="Layout_EventT">
       <item>
        <widget class="QFrame" name="EventT">
         <property name="frameShape">
          <enum>QFrame::StyledPanel</enum>
         </property>
         <layout class="QVBoxLayout" name="verticalLayout_3">
          <item>
           <widget class="QFrame" name="EventT_TitleFrame">
            <layout class="QHBoxLayout" name="horizontalLayout_2">
             <item>
              <widget class="QLabel" name="EventT_Title">
               <property name="text">
                <string>Weekly Schedule</string>
               </property>
              </widget>
             </item>
            </layout>
           </widget>
          </item>
          <item>
           <widget class="Line" name="line_3">
            <property name="orientation">
             <enum>Qt::Horizontal</enum>
            </property>
           </widget>
          </item>
          <item>
           <widget class="QTableWidget" name="EventT_Table">
            <row>
             <property name="text">
              <string>7:00 AM</string>
             </property>
            </row>
            <row>
             <property name="text">
              <string>8:00 AM</string>
             </property>
            </row>
            <row>
             <property name="text">
              <string>9:00 AM</string>
             </property>
            </row>
            <row>
             <property name="text">
              <string>10:00 AM</string>
             </property>
            </row>
            <row>
             <property name="text">
              <string>11:00 AM</string>
             </property>
            </row>
            <row>
             <property name="text">
              <string>12:00 PM</string>
             </property>
            </row>
            <row>
             <property name="text">
              <string>1:00 PM</string>
             </property>
            </row>
            <row>
             <property name="text">
              <string>2:00 PM</string>
             </property>
            </row>
            <row>
             <property name="text">
              <string>3:00 PM</string>
             </property>
            </row>
            <row>
             <property name="text">
              <string>4:00 PM</string>
             </property>
            </row>
            <row>
             <property name="text">
              <string>5:00AM</string>
             </property>
            </row>
            <row>
             <property name="text">
              <string>5:00 PM</string>
             </property>
            </row>
            <row>
             <property name="text">
              <string>7:00 PM</string>
             </property>
            </row>
            <column>
             <property name="text">
              <string>Sunday</string>
             </property>
            </column>
            <column>
             <property name="text">
              <string>Monday</string>
             </property>
            </column>
            <column>
             <property name="text">
              <string>Tuesday</string>
             </property>
            </column>
            <column>
             <property name="text">
              <string>Wednesday</string>
             </property>
            </column>
            <column>
             <property name="text">
              <string>Thursday</string>
             </property>
            </column>
            <column>
             <property name="text">
              <string>Friday</string>
             </property>
            </column>
            <column>
             <property name="text">
              <string>Saturday</string>
             </property>
            </column>
           </widget>
          </item>
         </layout>
        </widget>
       </item>
      </layout>
     </widget>
     <widget class="QWidget" name="horizontalLayoutWidget_4">
      <property name="geometry">
       <rect>
        <x>20</x>
        <y>30</y>
        <width>241</width>
        <height>461</height>
       </rect>
      </property>
      <layout class="QHBoxLayout" name="Layout_Events">
       <item>
        <widget class="QFrame" name="Events">
         <property name="frameShape">
          <enum>QFrame::StyledPanel</enum>
         </property>
         <layout class="QVBoxLayout" name="verticalLayout_4">
          <item>
           <widget class="QWidget" name="widget" native="true">
            <layout class="QHBoxLayout" name="horizontalLayout">
             <item alignment="Qt::AlignLeft">
              <widget class="QPushButton" name="pushButton_2">
               <property name="text">
                <string>Upcoming Events</string>
               </property>
              </widget>
             </item>
            </layout>
           </widget>
          </item>
          <item>
           <widget class="Line" name="line_4">
            <property name="orientation">
             <enum>Qt::Horizontal</enum>
            </property>
           </widget>
          </item>
          <item alignment="Qt::AlignLeft|Qt::AlignTop">
           <widget class="QLabel" name="DatePublished_Events">
            <property name="text">
             <string>Date Published</string>
            </property>
           </widget>
          </item>
          <item alignment="Qt::AlignHCenter">
           <widget class="QTableWidget" name="Events_table">
            <row>
             <property name="text">
              <string>7:00 AM</string>
             </property>
            </row>
            <row>
             <property name="text">
              <string>8:00 AM</string>
             </property>
            </row>
            <row>
             <property name="text">
              <string>9:00 AM</string>
             </property>
            </row>
            <row>
             <property name="text">
              <string>10:00 AM</string>
             </property>
            </row>
            <row>
             <property name="text">
              <string>11:00 AM</string>
             </property>
            </row>
            <row>
             <property name="text">
              <string>12:00 PM</string>
             </property>
            </row>
            <row>
             <property name="text">
              <string>1:00 PM</string>
             </property>
            </row>
            <row>
             <property name="text">
              <string>2:00 PM</string>
             </property>
            </row>
            <row>
             <property name="text">
              <string>3:00 PM</string>
             </property>
            </row>
            <row>
             <property name="text">
              <string>4:00 PM</string>
             </property>
            </row>
            <row>
             <property name="text">
              <string>5:00 PM</string>
             </property>
            </row>
            <row>
             <property name="text">
              <string>6:00 PM</string>
             </property>
            </row>
            <row>
             <property name="text">
              <string>7:00 PM</string>
             </property>
            </row>
            <column>
             <property name="text">
              <string/>
             </property>
            </column>
           </widget>
          </item>
          <item>
           <widget class="QPushButton" name="ViewAttendanceButton">
            <property name="text">
             <string>View Attendance</string>
            </property>
           </widget>
          </item>
         </layout>
        </widget>
       </item>
      </layout>
     </widget>
    </widget>
    <widget class="QWidget" name="page_2"/>
   </widget>
  </widget>
  <widget class="QStatusBar" name="statusbar"/>
 </widget>
 <resources/>
 <connections/>
</ui>
